### PR TITLE
add Ventus@0.2 w/ git auto-update

### LINKED ISSUE #8684: [Request] Add ventus
**Library name:** ventus
**Git repository url:** https://github.com/rlamana/Ventus
**npm package url(optional):** 
**License(s):**
**Official homepage:**  http://www.rlamana.es/ventus/
**Wanna say something? Leave message here:**
# 

Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #8684 
@pvnr0082t please help me review this pull request, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Profile of the lib
- Git repository (required): https://github.com/rlamana/Ventus
- Official website (optional, not the repository): http://www.rlamana.es/ventus/
- NPM package url (optional):
- GitHub / Bitbucket popularity (required):
  - Count of stars: 160
  - Count of watchers: 24
  - Count of forks: 45
- NPM download stats (optional):
  - Downloads in the last day:
  - Downloads in the last week:
  - Downloads in the last month:
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [x] This lib was not found on cdnjs repo
- [x] No already exist / duplicated issue and PR
- [x] The lib has notable popularity
  - [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [ ] More than 500 downloads stats per month on npm registry
- [ ] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [x] Separate unrelated changes into different commits.
- [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [x] Close corresponding issue in commit message
- [x] Mention related issue(s), people in commit message, comment.

close #8684, cc @rlamana

### CODE REVIEW

File: ajax/libs/Ventus/0.2/ventus.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+				} else if(style.webkitAnimation === '') {
+					event = 'webkitAnimationEnd';
+				}
+
+				return event;
+			}
+		},
+
+		WindowManager: require('ventus/wm/windowmanager'),
+		Window: require('ventus/wm/window')
```
Comment (pvnr0082t) [inline ajax/libs/Ventus/0.2/ventus.js]: @kennynaoh Can you confirm with author if we need other files? It seems like need some dependency files. I'm not sure.

Comment (PeterDaveHello) [inline ajax/libs/Ventus/0.2/ventus.js]: Looks it works well:
http://www.rlamana.es/ventus/code/examples/simple/

**Comment by PeterDaveHello** [comment]
My concern is the file you commited is not the same as its `dist` file in the example page, though they are both v0.2:
http://www.rlamana.es/ventus/code/dist/ventus.js

**Comment by kennynaoh** [comment]
@PeterDaveHello 
I will confirm with author, too.
Thanks!

**Comment by PeterDaveHello** [comment]
@kennynaoh Thank you!

**Comment by kennynaoh** [comment]
you're welcome

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by rlamana** [comment]
Hi, I'm the author of Ventus. As @PeterDaveHello pointed out, the files should be those from the `dist/`directory in Ventus repo (https://github.com/rlamana/Ventus/tree/v0.2.1/dist). No other dependencies needed besides those files.

Also, the official website is actually: [http://www.rlamana.com/ventus/](http://www.rlamana.com/ventus/) (not rlamana.es).

Thanks!

**Pull request closed without merging**